### DISCUSSION TITLE
Can Velero use local storage(persistent volume) for backup/restore

### DISCUSSION THREAD

**Discussion by andreacardoz22** [vmware-tanzu/velero#6249] [Community support Q&A]
Hi Team,

Can you please let me know if it is possible to use local storage(persistent volume) as a backup storage location for backup and restore operations? If not, is this going to be made available in the near future?

**Comment by draghuram** [comment]
Velero can only use  S3 for backup storage location at present and I don't think there are any plans of  supporting file system target. Having said that, you may consider running a S3 service (such as MinIO) backed by your local storage and then configure that S3 endpoint in Velero.

Also, in case you are thinking of using the PVs in the same cluster that is being backed up, that is not a good idea. Backups should be stored separate from the backup sources.
